contribs/osm/src/main/java/org/matsim/contrib/osm/networkReader/SupersonicOsmNetworkReader.java

```diff
@@ -196,7 +196,7 @@ Link createLink(Node fromNode, Node toNode, WaySegment segment, Direction direct
         link.setLength(segment.getLength());
         link.setFreespeed(getFreespeed(segment.getTags(), link.getLength(), properties));
```

Comment:
Maybe, for consistency, it would be better to move the other methods, such as getFreespeed, getNumberOfLanes that extract link properties into the LinkProperties class?

The way it is now, we have some guessing of properties in the reader and some in the LinkProperties class

Reply:
I was thinking these other methods depends heavily on OSM-Tags, so they would not be so useful outside this context. I don't think they would be likely to be used, so I left them there.

Reply:
Okay, that makes sence.